src/Client.php

```diff
@@ -173,14 +173,18 @@ public function captureEvent(Event $event, ?EventHint $hint = null, ?Scope $scop
         }
 
         try {
-            /** @var Response $response */
-            $response = $this->transport->send($event)->wait();
-            $event = $response->getEvent();
+            /** @var Result $result */
+            $result = $this->transport->send($event);
+            $event = $result->getEvent();
```

Comment:
We might also want to wrap this in a `trace()` so requests to Sentry are logged as a span in case errors are transmitted inside a performance traced request (or maybe that should be done inside the Transport or somewhere else but it would be nice to have a span for it.

Reply:
Okay that would make sense, although it also doesn't 😅 Interestring to know why since it could account for a large-ish gap in the spans if the app sends an error and the connection is terrible for some reason. Would be great to know (and nice insights maybe for Sentry data-miners).

Reply:
TBD, will get back later.